ArtOfIllusion/src/artofillusion/ScrollViewTool.java

```diff
@@ -0,0 +1,301 @@
+/* Copyright (C) 2017 by Petri Ihalainen
```

Comment:
Hmm... I think that `MoveViewTool` and `RotateViewTool` are separate because they are being registered as `EditingTools` with the toolpalette.

For the moment, we'll leave it. Let's get the rebase done first.